config/routes.rb

```diff
@@ -1,5 +1,5 @@
 MANIACChess::Application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
There's no need to add a Users controller. I was referring to `OmniauthCallbacksController`. With this route config, the app will expect it to be located under app/controllers/users and to be part of the `User::` namespace. You should either update routes.rb (seems easier) or update the controller and its associated helpers, specs, etc. More info here: http://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing